rasa/nlu/registry.py

```diff
@@ -96,6 +96,15 @@
 def get_component_class(component_name: Text) -> Type["Component"]:
     """Resolve component name to a registered components class."""
 
+    if component_name == "DucklingHTTPExtractor":
+        raise_warning(
```

Comment:
should we use `raise_deprecation_warning()`?

Reply:
```suggestion
        raise_deprecation_warning(
```